src/common/ast-path.js

```diff
@@ -48,6 +74,17 @@ class AstPath {
     return getNodeHelper(this, count + 1);
   }
 
+  #getSiblingNode(offset) {
+    const { stack } = this;
+    const { length } = stack;
+    assert(length < 3);
```

Comment:
Shouldn't this at least `5`?

Reply:
IDK. Can't the root of the AST be an array?

Reply:
Okay, changed it to 5 to make the check stricter.